src/formatDate.js

```diff
@@ -8,7 +8,30 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
```

Comment:
The separator is assumed to be at index 3 of the `fromFormat` array. Ensure that this assumption is valid for all expected input formats. If the separator is not always at this position, the logic may need to be adjusted.